recipes/wasmer/all/conandata.yml

```diff
@@ -0,0 +1,17 @@
+sources:
+  "2.0.0":
+    Windows:
+      "x86_64":
+        url: "https://github.com/wasmerio/wasmer/releases/download/2.0.0/wasmer-windows-amd64.tar.gz"
```

Comment:
I don't think this contains a static library.
`wasmer.dll` is the binary, and `wasmer.lib` is the import library.
The archive does not contain a stand-alone static library.

Reply:
If needed, you can create an import library using `pexports` (we have a recipe for that.
I would create an issue at the wasmer repo.
I did the same thing for wasmtime and they fixed it in the next release.